components/nlus/nlu.ts

```diff
@@ -103,8 +104,21 @@ export default class Nlu extends Component {
         }
     }
 
+    public async pull() {
+        const projectId = this.helper.getProp("projectId");
+        try {
+            const { response: { body } } = await this.helper.toPromise(this.api.projectApi, this.api.projectApi.projectsProjectIdNluGet, projectId);
```

Comment:
ini yang bener `this.api.projectApi.projectsProjectIdNluGet` atau `this.api.nluApi.projectsProjectIdNluGet` @rilut ?

Reply:
saya lihat di zaun swaggernya sudah bener di yg `projectApi` mba deww